ibis/expr/operations.py

```diff
@@ -1025,6 +1025,17 @@ def __init__(self, expr, window):
         if table is not None:
             window = window.bind(table)
 
+        if window.max_lookback is not None:
+            if len(window._order_by) != 1:
+                raise com.IbisInputError(
+                    "'max lookback windows must be ordered by time"
+                )
+            order_var = window._order_by[0].op().args[0]
+            if not isinstance(order_var._dtype, dt.Timestamp):
+                raise com.IbisInputError(
+                    "'max lookback' windows must be ordered by time"
```

Comment:
Let's make this `"... ordered by a timestamp column"` so that there's no ambiguity about whether it's the type or the column name that's the problem.

Reply:
Fixed in next commit